Incorrect info in tests/README.rst

The tests/README.rst document says:
<pre>
Autospec has limited functional testing capabilities that can be invoked by running make test_autospec from the root of the autospec directory
</pre>

However, the target test_autospec was removed from the root Makefile a long time ago (1/10/2018). So "make test_autospec" from the root of the autospec directory reports:
<pre>
make: *** No rule to make target 'test_autospec'.  Stop.
<_pre>

--- a/tests/README.rst
+++ b/tests/README.rst
@@ -2,76 +2,22 @@
 Autospec Testing
 ================
 
-Functional
+Code Style
 ==========
 
-Autospec has limited functional testing capabilities that can be invoked by
-running ``make test_autospec`` from the root of the autospec directory.
-
-The tests first create a tarball of each source directory within
-``tests/testfiles/`` and run autospec against each. Each package is handled in
-its own thread, so the total time of the test run is equivalent to the total
-time of the longest autospec run.
-
-Once the build is complete, the output of autospec and the resulting spec file
-are searched for terms defined in a special ``expectations.py`` file within the
-test source directories. Several terms can be defined in ``expectations.py``
-which determine what will be tested after a successful autospec run.
-
-- buildreqs (list of strings) - build requirements expected to be detected by
-  autospec.
-
-- license (string) - name of the license to be detected by autospec.
-
-- output_strings (list of strings) - strings expected to be in the output of the
-  autospec run.
-
-- specfile (string) - this string is read in from the ``spec-expectations``
-  file, which contains the exact text of the specfile expected to be generated
-  from the autospec run. If there are discrepancies between the generated and
-  expected spec files a diff will be displayed in the results.
-
-- conffile (string) - this string is read in from the ``conf-expectations``
-  file, which contains the exact text of the ``options.conf`` file expected to
-  be generated (or changed, or left unchanged) from the autospec run. If there
-  are discrepancies between the generated and expected ``options.conf`` files a
-  diff will be displayed in the results.
-
-There is also a directory within the source file directory called
-``autospecdir`` that contains the package-specific configuration files used by
-the autospec run. Here you can set specific configuration files such as the
-deprecated boolean configs for testing ``options.conf`` generation, or
-``buildreq_add`` to test build requirement detection.
-
-If an autospec build fails for some reason, the build log will be placed in
-``tests/testfiles/results/<pkgname>.log``. Any old logs are removed at the
-beginning of a test run to avoid any indication that a build failed when it did
-not.
-
-The overall test directory structure looks like this:
-
-::
-
-  tests/
-      test_autospec.py     # functional tests
-      pkg_integrity.py     # unit tests
-      testfiles/
-          <source1>/
-              expectations.py
-              spec-expectations
-              conf-expectations
-              <source files>
-              autospecdir/
-                  <configuration files for autospec>
-          <source2>/
-              ...
-          ...
-          results/
-              <may contain failed build log>
-
+Autospec changes are scanned to vet code style issues with the ``flake8`` tool.
+To check for issues, run ``make check`` from the root of the autospec source
+tree, which executes ``flake8`` with appropriate arguments.
 
 Unit
 ====
 
-One autospec module, ``pkg_integrity.py`` also has unit testing available via
-``make test_pkg_integrity``.
+Autospec ships with several test modules that correspond to individual modules
+from the toplevel ``autospec`` directory.
+
+Each module can be tested in isolation by running ``make test_<MODULE>``, where
+``<MODULE>`` corresponds to the module name. For example, ``make
+test_pkg_integrity`` runs unit tests for the ``pkg_integrity.py`` module.
+
+To run *all* unit tests, run ``make unittests``. If all tests pass, a code
+coverage report is also generated.

